credential-controller.js

```diff
@@ -159,6 +159,9 @@ function factory(
         if(!('identity' in session) && self.credential.sysIsPublic) {
           // display a public credential to a non-authenticated user
           _display('credentialInfo');
+        } else if('CREDENTIAL_ADMIN' in session.identity.sysPermissionTable) {
```

Comment:
Can we change this to:

```js
} else if('sysPermissionTable' in session.identity &&
  'CREDENTIAL_ADMIN' in session.identity.sysPermissionTable) {
```

Just so things don't break during transition to the new feature.